src/projectBank2/Account.java

```diff
@@ -0,0 +1,77 @@
+package projectBank2;

```

Comment:
Use proper package e.g com.swiderski.bank. Also do not use captial cases and avoid numbers. I know it is only working copy, but it is good to get those good practices from the beging.

Reply:
Test